src/pageObject.js

```diff
@@ -1,15 +1,39 @@
 'use strict';
 
 class Header {
-
+  get logo() {
+    return 'Conduit logo';
+  }
 };
 
 class PageObject {
+constructor(url,header){
+  this.url = url;
+  this.header = new Header();
+}
 
+clickOnLogo() {
+  return 'Click on the ' + this.header.logo;
+}
 };
 
 class ArticlePage extends PageObject {
+  constructor(url, slug) {
+    super(url);
```

Comment:
The super() call in the ArticlePage constructor should pass both 'url' and 'header' if the PageObject constructor requires two arguments. Currently, it only passes 'url', which might lead to unexpected behavior if 'header' is needed.